libherokubuildpack/Cargo.toml

```diff
@@ -50,8 +50,9 @@ tar = { version = "0.4.44", default-features = false, optional = true }
 termcolor = { version = "1.4.1", optional = true }
 thiserror = { version = "2.0.12", optional = true }
 toml = { workspace = true, optional = true }
-ureq = { version = "3.0.12", default-features = false, features = ["rustls"], optional = true }
+reqwest = { version = "0.12.23", features = ["blocking", "rustls-tls-native-roots"], optional = true }
```

Comment:
> I could not get it to work with `ureq` at this point.

Could you expand on this?

Was there a bug with ureq that meant native TLS wasn't being honoured? If so, should we file something upstream?

I'm fine with landing this as a stop-gap in the meantime prior to writing up a bug report - but we should definitely file a bug if there is one (and there's no explanation as to the rough reason so far in the PR?)